tests/D2L.CodeStyle.Analyzers.Test/Specs/ConstantAnalyzer.cs

```diff
@@ -14,11 +14,16 @@ namespace SpecTests
 {
 	using D2L.CodeStyle.Annotations.Contract;
 
-	public sealed class Logger
-	{
+	public sealed class Logger {
 		public static void Error( [Constant] string message ) { }
 	}
 
+	public sealed class WrappedLogger {
+		public static void Error( [Constant] string message ) {
+			Logger.Error( message );
+		}
+	}

```

Comment:
Add a negative test case showing a non-[Constant] parameter tripping the analyzer?